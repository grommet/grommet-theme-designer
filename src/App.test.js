import React from 'react';
import { cleanup, render, fireEvent } from '@testing-library/react';
import App from './App';

describe('App', () => {
  afterEach(cleanup);

  test('empty', () => {
    const { getByLabelText, container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();

    fireEvent.click(getByLabelText('show my themes'));
    expect(container.firstChild).toMatchSnapshot();
  });
});
