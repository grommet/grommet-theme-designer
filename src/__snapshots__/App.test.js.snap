// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App empty 1`] = `
<div
  class="StyledGrommet-sc-19lkkz7-0 dAyrEz"
>
  <div
    class="StyledGrid-sc-1wofa1l-0 BUNVq"
  >
    <div
      class="StyledBox-sc-13pk1d4-0 msHTX"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 hKtyGr"
      >
        <span
          class="StyledText-sc-1sadyjn-0 onXHw"
        >
          my theme
        </span>
        <button
          class="StyledButton-sc-323bzc-0 eyZaup"
          title="edit theme"
          type="button"
        >
          <svg
            aria-label="Edit"
            class="StyledIcon-ofa7kd-0 htxtFB"
            viewBox="0 0 24 24"
          >
            <path
              d="m14 4 6 6-6-6zm8.294 1.294c.39.39.387 1.025-.008 1.42L9 20l-7 2 2-7L17.286 1.714a1 1 0 0 1 1.42-.008l3.588 3.588zM3 19l2 2m2-4 8-8"
              fill="none"
              stroke="#000"
              stroke-width="2"
            />
          </svg>
        </button>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 jcBJPm"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 kUTqHn"
          role="radiogroup"
        >
          <label
            class="StyledRadioButton__StyledRadioButtonContainer-g1f6ld-0 jBepmx"
            for="light"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 cWAgDf StyledRadioButton-g1f6ld-5 cQxxvY"
            >
              <input
                checked=""
                class="StyledRadioButton__StyledRadioButtonInput-g1f6ld-1 bRmCoz"
                id="light"
                name="themeMode"
                tabindex="0"
                type="radio"
                value="light"
              />
              <div
                class="StyledBox-sc-13pk1d4-0 jdZBB StyledRadioButton__StyledRadioButtonBox-g1f6ld-4 lkJviK"
              >
                <svg
                  class="StyledRadioButton__StyledRadioButtonIcon-g1f6ld-3 dwAXjB"
                  preserveAspectRatio="xMidYMid meet"
                  viewBox="0 0 24 24"
                >
                  <circle
                    cx="12"
                    cy="12"
                    r="6"
                  />
                </svg>
              </div>
            </div>
            <span
              class="StyledRadioButton__StyledRadioButtonLabel-g1f6ld-2"
            >
              light
            </span>
          </label>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 htfuze"
          />
          <label
            class="StyledRadioButton__StyledRadioButtonContainer-g1f6ld-0 jBepmx"
            for="dark"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 cWAgDf StyledRadioButton-g1f6ld-5 cQxxvY"
            >
              <input
                class="StyledRadioButton__StyledRadioButtonInput-g1f6ld-1 bRmCoz"
                id="dark"
                name="themeMode"
                tabindex="-1"
                type="radio"
                value="dark"
              />
              <div
                class="StyledBox-sc-13pk1d4-0 hgnDLt StyledRadioButton__StyledRadioButtonBox-g1f6ld-4 lkJviK"
              />
            </div>
            <span
              class="StyledRadioButton__StyledRadioButtonLabel-g1f6ld-2"
            >
              dark
            </span>
          </label>
        </div>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 qEKCj"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 jcBJPm"
        >
          <h3
            class="StyledHeading-sc-1rdh4aw-0 eSDDJo"
          >
            Guides
          </h3>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Accessibility
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Typography
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 jcBJPm"
        >
          <h3
            class="StyledHeading-sc-1rdh4aw-0 eSDDJo"
          >
            Examples
          </h3>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Dashboard
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          List
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Form
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Document
        </div>
      </div>
    </div>
    <div
      class="StyledBox-sc-13pk1d4-0 kUTqHn"
    />
    <div
      class="StyledBox-sc-13pk1d4-0 kRkAyf"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 fKqrK"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 bUZGIl"
        >
          <button
            class="StyledButton-sc-323bzc-0 eyZaup"
            title="choose another theme"
            type="button"
          >
            <svg
              aria-label="Apps"
              class="StyledIcon-ofa7kd-0 htxtFB"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 5h2V3h-2v2zm-8 0h2V3h-2v2zM3 5h2V3H3v2zm16 8h2v-2h-2v2zm-8 0h2v-2h-2v2zm-8 0h2v-2H3v2zm16 8h2v-2h-2v2zm-8 0h2v-2h-2v2zm-8 0h2v-2H3v2z"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 jMNBey"
          />
          <div
            class="StyledBox-sc-13pk1d4-0 dnSXwv"
          >
            <button
              class="StyledButton-sc-323bzc-0 iXpptv"
              disabled=""
              title="undo last change"
              type="button"
            >
              <svg
                aria-label="Undo"
                class="StyledIcon-ofa7kd-0 htxtFB"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7.18 4 8.6 5.44 6.06 8h9.71a6 6 0 0 1 0 12h-2v-2h2a4 4 0 0 0 0-8H6.06l2.54 2.51-1.42 1.41L2.23 9z"
                />
              </svg>
            </button>
            <button
              class="StyledButton-sc-323bzc-0 iXpptv"
              disabled=""
              title="redo last change"
              type="button"
            >
              <svg
                aria-label="Redo"
                class="StyledIcon-ofa7kd-0 htxtFB"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.82 4 15.4 5.44 17.94 8H8.23a6 6 0 0 0 0 12h2v-2h-2a4 4 0 0 1 0-8h9.71l-2.54 2.51 1.41 1.41L21.77 9z"
                />
              </svg>
            </button>
          </div>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 jMNBey"
          />
          <button
            class="StyledButton-sc-323bzc-0 eyZaup"
            title="see JSON"
            type="button"
          >
            <svg
              aria-label="Code"
              class="StyledIcon-ofa7kd-0 htxtFB"
              viewBox="0 0 24 24"
            >
              <path
                d="m9 22 6-20m2 15 5-5-5-5M7 17l-5-5 5-5"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 jMNBey"
          />
          <button
            class="StyledButton-sc-323bzc-0 eyZaup"
            title="share"
            type="button"
          >
            <svg
              aria-label="Share"
              class="StyledIcon-ofa7kd-0 htxtFB"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13v10H1V5h10m3-4h9v9m-13 4L23 1 10 14z"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 laRhbc"
        >
          <div
            class="StyledBox-sc-13pk1d4-0 dWtskw"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 gNbYdS"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 erlESz"
                for="name"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 fxbXHC"
                >
                  name
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 kUTqHn"
              >
                <div
                  class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 gXHnix"
                >
                  <input
                    autocomplete="off"
                    class="StyledTextInput-sc-1x30a0s-0 dsEprr"
                    name="name"
                    style="text-align: right;"
                    value="my theme"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dWtskw"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 gNbYdS"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 erlESz"
                for="rounding"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 fxbXHC"
                >
                  rounding
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 kUTqHn"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 eywysP"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 bYcekd"
                    id="rounding"
                    max="24"
                    min="0"
                    name="rounding"
                    step="2"
                    style="text-align: end;"
                    type="range"
                    value="4"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 bXjkrf"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 foCyPH"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 gHuPht"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="4"
                      />
                    </div>
                    <span
                      class="StyledText-sc-1sadyjn-0 fxbXHC"
                    >
                      px
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dWtskw"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 gNbYdS"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 erlESz"
                for="spacing"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 fxbXHC"
                >
                  spacing
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 kUTqHn"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 eywysP"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 bYcekd"
                    id="spacing"
                    max="36"
                    min="12"
                    name="spacing"
                    step="4"
                    style="text-align: end;"
                    type="range"
                    value="24"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 bXjkrf"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 foCyPH"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 gHuPht"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="24"
                      />
                    </div>
                    <span
                      class="StyledText-sc-1sadyjn-0 fxbXHC"
                    >
                      px
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dWtskw"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 gNbYdS"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 erlESz"
                for="scale"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 fxbXHC"
                >
                  font scale
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 kUTqHn"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 eywysP"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 bYcekd"
                    id="scale"
                    max="4"
                    min="0.2"
                    name="scale"
                    step="0.1"
                    style="text-align: end;"
                    type="range"
                    value="1"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 bXjkrf"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 foCyPH"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 gHuPht"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="1"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <button
            aria-label="Open Drop"
            class="StyledButton-sc-323bzc-0 iRgBrf"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 eryjQU"
            >
              <span
                class="StyledText-sc-1sadyjn-0 fxbXHC"
              >
                font
              </span>
              <div
                class="StyledBox__StyledBoxGap-sc-13pk1d4-1 jMNBey"
              />
              <span
                class="StyledText-sc-1sadyjn-0 onXHw"
              >
                Helvetica
              </span>
            </div>
          </button>
          <button
            class="StyledButton-sc-323bzc-0 iRgBrf"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 eryjQU"
            >
              <span
                class="StyledText-sc-1sadyjn-0 fxbXHC"
              >
                colors
              </span>
            </div>
          </button>
          <button
            class="StyledButton-sc-323bzc-0 iRgBrf"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 eryjQU"
            >
              <span
                class="StyledText-sc-1sadyjn-0 fxbXHC"
              >
                form field
              </span>
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`App empty 2`] = `
<div
  class="StyledGrommet-sc-19lkkz7-0 dAyrEz"
>
  <div
    class="StyledGrid-sc-1wofa1l-0 BUNVq"
  >
    <div
      class="StyledBox-sc-13pk1d4-0 msHTX"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 hKtyGr"
      >
        <span
          class="StyledText-sc-1sadyjn-0 onXHw"
        >
          my theme
        </span>
        <button
          class="StyledButton-sc-323bzc-0 eyZaup"
          title="edit theme"
          type="button"
        >
          <svg
            aria-label="Edit"
            class="StyledIcon-ofa7kd-0 htxtFB"
            viewBox="0 0 24 24"
          >
            <path
              d="m14 4 6 6-6-6zm8.294 1.294c.39.39.387 1.025-.008 1.42L9 20l-7 2 2-7L17.286 1.714a1 1 0 0 1 1.42-.008l3.588 3.588zM3 19l2 2m2-4 8-8"
              fill="none"
              stroke="#000"
              stroke-width="2"
            />
          </svg>
        </button>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 jcBJPm"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 kUTqHn"
          role="radiogroup"
        >
          <label
            class="StyledRadioButton__StyledRadioButtonContainer-g1f6ld-0 jBepmx"
            for="light"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 cWAgDf StyledRadioButton-g1f6ld-5 cQxxvY"
            >
              <input
                checked=""
                class="StyledRadioButton__StyledRadioButtonInput-g1f6ld-1 bRmCoz"
                id="light"
                name="themeMode"
                tabindex="0"
                type="radio"
                value="light"
              />
              <div
                class="StyledBox-sc-13pk1d4-0 jdZBB StyledRadioButton__StyledRadioButtonBox-g1f6ld-4 lkJviK"
              >
                <svg
                  class="StyledRadioButton__StyledRadioButtonIcon-g1f6ld-3 dwAXjB"
                  preserveAspectRatio="xMidYMid meet"
                  viewBox="0 0 24 24"
                >
                  <circle
                    cx="12"
                    cy="12"
                    r="6"
                  />
                </svg>
              </div>
            </div>
            <span
              class="StyledRadioButton__StyledRadioButtonLabel-g1f6ld-2"
            >
              light
            </span>
          </label>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 htfuze"
          />
          <label
            class="StyledRadioButton__StyledRadioButtonContainer-g1f6ld-0 jBepmx"
            for="dark"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 cWAgDf StyledRadioButton-g1f6ld-5 cQxxvY"
            >
              <input
                class="StyledRadioButton__StyledRadioButtonInput-g1f6ld-1 bRmCoz"
                id="dark"
                name="themeMode"
                tabindex="-1"
                type="radio"
                value="dark"
              />
              <div
                class="StyledBox-sc-13pk1d4-0 hgnDLt StyledRadioButton__StyledRadioButtonBox-g1f6ld-4 lkJviK"
              />
            </div>
            <span
              class="StyledRadioButton__StyledRadioButtonLabel-g1f6ld-2"
            >
              dark
            </span>
          </label>
        </div>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 qEKCj"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 jcBJPm"
        >
          <h3
            class="StyledHeading-sc-1rdh4aw-0 eSDDJo"
          >
            Guides
          </h3>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Accessibility
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Typography
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 jcBJPm"
        >
          <h3
            class="StyledHeading-sc-1rdh4aw-0 eSDDJo"
          >
            Examples
          </h3>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Dashboard
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          List
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Form
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 dPHCeG"
          tabindex="0"
        >
          Document
        </div>
      </div>
    </div>
    <div
      class="StyledBox-sc-13pk1d4-0 kUTqHn"
    />
    <div
      class="StyledBox-sc-13pk1d4-0 kRkAyf"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 fKqrK"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 bUZGIl"
        >
          <button
            class="StyledButton-sc-323bzc-0 eyZaup"
            title="choose another theme"
            type="button"
          >
            <svg
              aria-label="Apps"
              class="StyledIcon-ofa7kd-0 htxtFB"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 5h2V3h-2v2zm-8 0h2V3h-2v2zM3 5h2V3H3v2zm16 8h2v-2h-2v2zm-8 0h2v-2h-2v2zm-8 0h2v-2H3v2zm16 8h2v-2h-2v2zm-8 0h2v-2h-2v2zm-8 0h2v-2H3v2z"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 jMNBey"
          />
          <div
            class="StyledBox-sc-13pk1d4-0 dnSXwv"
          >
            <button
              class="StyledButton-sc-323bzc-0 iXpptv"
              disabled=""
              title="undo last change"
              type="button"
            >
              <svg
                aria-label="Undo"
                class="StyledIcon-ofa7kd-0 htxtFB"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7.18 4 8.6 5.44 6.06 8h9.71a6 6 0 0 1 0 12h-2v-2h2a4 4 0 0 0 0-8H6.06l2.54 2.51-1.42 1.41L2.23 9z"
                />
              </svg>
            </button>
            <button
              class="StyledButton-sc-323bzc-0 iXpptv"
              disabled=""
              title="redo last change"
              type="button"
            >
              <svg
                aria-label="Redo"
                class="StyledIcon-ofa7kd-0 htxtFB"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.82 4 15.4 5.44 17.94 8H8.23a6 6 0 0 0 0 12h2v-2h-2a4 4 0 0 1 0-8h9.71l-2.54 2.51 1.41 1.41L21.77 9z"
                />
              </svg>
            </button>
          </div>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 jMNBey"
          />
          <button
            class="StyledButton-sc-323bzc-0 eyZaup"
            title="see JSON"
            type="button"
          >
            <svg
              aria-label="Code"
              class="StyledIcon-ofa7kd-0 htxtFB"
              viewBox="0 0 24 24"
            >
              <path
                d="m9 22 6-20m2 15 5-5-5-5M7 17l-5-5 5-5"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 jMNBey"
          />
          <button
            class="StyledButton-sc-323bzc-0 eyZaup"
            title="share"
            type="button"
          >
            <svg
              aria-label="Share"
              class="StyledIcon-ofa7kd-0 htxtFB"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13v10H1V5h10m3-4h9v9m-13 4L23 1 10 14z"
                fill="none"
                stroke="#000"
                stroke-width="2"
              />
            </svg>
          </button>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 laRhbc"
        >
          <div
            class="StyledBox-sc-13pk1d4-0 dWtskw"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 gNbYdS"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 erlESz"
                for="name"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 fxbXHC"
                >
                  name
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 kUTqHn"
              >
                <div
                  class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 gXHnix"
                >
                  <input
                    autocomplete="off"
                    class="StyledTextInput-sc-1x30a0s-0 dsEprr"
                    name="name"
                    style="text-align: right;"
                    value="my theme"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dWtskw"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 gNbYdS"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 erlESz"
                for="rounding"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 fxbXHC"
                >
                  rounding
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 kUTqHn"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 eywysP"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 bYcekd"
                    id="rounding"
                    max="24"
                    min="0"
                    name="rounding"
                    step="2"
                    style="text-align: end;"
                    type="range"
                    value="4"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 bXjkrf"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 foCyPH"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 gHuPht"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="4"
                      />
                    </div>
                    <span
                      class="StyledText-sc-1sadyjn-0 fxbXHC"
                    >
                      px
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dWtskw"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 gNbYdS"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 erlESz"
                for="spacing"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 fxbXHC"
                >
                  spacing
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 kUTqHn"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 eywysP"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 bYcekd"
                    id="spacing"
                    max="36"
                    min="12"
                    name="spacing"
                    step="4"
                    style="text-align: end;"
                    type="range"
                    value="24"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 bXjkrf"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 foCyPH"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 gHuPht"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="24"
                      />
                    </div>
                    <span
                      class="StyledText-sc-1sadyjn-0 fxbXHC"
                    >
                      px
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dWtskw"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 gNbYdS"
            >
              <label
                class="StyledBox-sc-13pk1d4-0 erlESz"
                for="scale"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 fxbXHC"
                >
                  font scale
                </span>
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 kUTqHn"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 eywysP"
                >
                  <input
                    class="StyledRangeInput-sc-15st9ck-0 bYcekd"
                    id="scale"
                    max="4"
                    min="0.2"
                    name="scale"
                    step="0.1"
                    style="text-align: end;"
                    type="range"
                    value="1"
                  />
                  <div
                    class="StyledBox-sc-13pk1d4-0 bXjkrf"
                  >
                    <div
                      class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 foCyPH"
                    >
                      <input
                        autocomplete="off"
                        class="StyledMaskedInput-sc-99vkfa-0 gHuPht"
                        placeholder=""
                        style="text-align: right; width: 48px;"
                        value="1"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <button
            aria-label="Open Drop"
            class="StyledButton-sc-323bzc-0 iRgBrf"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 eryjQU"
            >
              <span
                class="StyledText-sc-1sadyjn-0 fxbXHC"
              >
                font
              </span>
              <div
                class="StyledBox__StyledBoxGap-sc-13pk1d4-1 jMNBey"
              />
              <span
                class="StyledText-sc-1sadyjn-0 onXHw"
              >
                Helvetica
              </span>
            </div>
          </button>
          <button
            class="StyledButton-sc-323bzc-0 iRgBrf"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 eryjQU"
            >
              <span
                class="StyledText-sc-1sadyjn-0 fxbXHC"
              >
                colors
              </span>
            </div>
          </button>
          <button
            class="StyledButton-sc-323bzc-0 iRgBrf"
            type="button"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 eryjQU"
            >
              <span
                class="StyledText-sc-1sadyjn-0 fxbXHC"
              >
                form field
              </span>
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
